Simplified/MyBooks/NYPLMyBooksDownloadCenter+AudiobooksDownload.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  NYPLMyBooksDownloadCenter+AudiobooksDownload.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2022-04-06.
+//  Copyright © 2022 NYPL. All rights reserved.
+//
+#if FEATURE_AUDIOBOOKS
+import Foundation
+import NYPLAudiobookToolkit
+
+extension NYPLMyBooksDownloadCenter: NYPLAudiobookDownloadStatusDelegate {
+  func audiobookDidUpdateDownloadProgress(progress: Float, bookID: String) {
+    downloadProgressDidUpdate(to: Double(progress), forBookIdentifier: bookID)
+  }
+  
+  func audiobookDidCompleteDownload(bookID: String) {
+    NYPLBookRegistry.shared().setStateWithCode(NYPLBookState.DownloadSuccessful.rawValue,
```

Comment:
can you pass in the BookRegistry as a protocol instead?

Reply:
The reason I didn't pass it in as a protocol is because this method comes from `NYPLAudiobookDownloader`, which does not use `NYPLBookRegistry` at all. With the NYPLMyBooksDownloadCenter refactor, I am expecting the `NYPLBookRegistry` singleton will be passed in as a protocol and stored as a property of the download center, given the amount it is being called. And we will be able to call something like `self.bookRegistryProvider` instead of the singleton after the refactoring. Let me know what do you think about it.

Reply:
yeah, good point. The audiobook downloader probably shouldn't even know about the bookRegistry. On the other hand, my plan is to not have a "Download Center" abstraction in the MyBooks refactor ;) In any case, you're right, let's not worry about it now since we're going to rewrite the DownloadCenter anyway.